website/docs/index.html.markdown

```diff
@@ -68,8 +68,13 @@ resource "newrelic_alert_policy_channel" "alert_email" {
 
 The following arguments are supported:
 
-- `api_key` - (Required) Your New Relic API key. Can also use `NEWRELIC_API_KEY` environment variable.
+- `api_key` - (Optional) Your New Relic API key. Can also use `NEWRELIC_API_KEY` environment variable.
 - `api_url` - (Optional) Possibility to change the API url (default is https://api.newrelic.com/v2). This is for instance relevant if the New Relic account is in the EU. The API url must then be set to https://api.eu.newrelic.com/v2. Can also use `NEWRELIC_API_URL` environment variable.
 - `infra_api_url` - (Optional) Possibility to change the Infra API url (default is https://infra-api.newrelic.com/v2). This is for instance relevant if the New Relic account is in the EU. The Infra API url must then be set to https://infra-api.eu.newrelic.com/v2. Can also use `NEWRELIC_INFRA_API_URL` environment variable.
 - `insecure_skip_verify` - (Optional) Trust self-signed SSL certificates. If omitted, the `NEWRELIC_API_SKIP_VERIFY` environment variable is used.
+- `insights_account_id` - (Optional) Your New Relic Account ID used when inserting or querying Insights events. Can also use `NEWRELIC_INSIGHTS_ACCOUNT_ID` environment variable.
```

Comment:
Can we change the wording of these to reference the actual resource name here (`newrelic_insights_event`)?